cpp/include/cudf/groupby.hpp

```diff
@@ -85,12 +85,12 @@ class groupby {
    * @param ignore_null_keys Indicates whether rows in `keys` that contain NULL
    * values should be ignored
    * @param keys_are_sorted Indicates whether rows in `keys` are already sorted
-   * @param column_order If `keys_are_sorted == true`, indicates whether each
+   * @param column_order If @p `keys_are_sorted == true`, indicates whether each
```

Comment:
I don't think you should mix markdown and @p like this. I don't think @p does anything other than use a typewriter font (no linking) on the next word, so I think MarkDown is clearer.

```suggestion
   * @param column_order If `keys_are_sorted == true`, indicates whether each
```

Reply:
I had noticed that my Doxygen plugin for VSCode highlighted words that came immediately after `@p`. I thought this might mean Doxygen might actually do something special with the `@p` different from just using the markdown backticks. However, looking at the generated HTML, you are correct that `@p` has no different effect than using back ticks. I think we should standardize on the backticks then. 
![Screenshot from 2019-11-27 10-25-46](https://user-images.githubusercontent.com/15221289/69741258-5ca1c800-1100-11ea-8eb8-7a2391ab6d1a.png)